astropy/utils/iers/tests/test_iers.py

```diff
@@ -251,6 +251,7 @@ def test_no_auto_download(self):
         assert type(t) is iers.IERS_B
 
     @pytest.mark.remote_data
+    @pytest.mark.xfail(reason='See issue 9600')
```

Comment:
I marked this whole test as xfail until #9600 is fixed. Also reverted my other test modifications to this file, @astrofrog .